src/main/java/ticTacToe/guiApp/ButtonsCreator.java

```diff
@@ -0,0 +1,32 @@
+package ticTacToe.guiApp;
+
+import javafx.scene.control.Button;
+import ticTacToe.grid.Grid;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ButtonsCreator {
+
+    private final ActionSetter actionSetter;
+    private List<ButtonWrapper> buttonsCreated;
+
+    public ButtonsCreator(ActionSetter actionSetter) {
+        this.actionSetter = actionSetter;
+    }
+
+    public List<ButtonWrapper> createButtons(Grid grid) {
+        buttonsCreated = new ArrayList<>();
+        for (String cellNumber : grid.getCells()) {
+            buttonsCreated.add(new JavaFxButton(new Button(), cellNumber));
+        }
+        setActionToButtons();
+        return buttonsCreated;
+    }
+
+    private void setActionToButtons() {
+        for (ButtonWrapper button : buttonsCreated) {
+            actionSetter.addClickHandler(button, button.actualButton().getText());
```

Comment:
How come I need to give the clickHandler both the `button` aaaaaand the text _on_ the button?
What is the _intent_ of the ClickHandler?